Formula/rtmpdump.rb

```diff
@@ -1,10 +1,10 @@
 class Rtmpdump < Formula
   desc "Tool for downloading RTMP streaming media"
   homepage "https://rtmpdump.mplayerhq.hu"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/r/rtmpdump/rtmpdump_2.4+20151223.gitfa8646d.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/r/rtmpdump/rtmpdump_2.4%2b20151223.gitfa8646d.orig.tar.gz"
+  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/r/rtmpdump/rtmpdump_2.4+20151223.gitfa8646d.1.orig.tar.gz"
+  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/r/rtmpdump/rtmpdump_2.4%2b20151223.gitfa8646d.1.orig.tar.gz"
   version "2.4+20151223"
```

Comment:
If I'm not misremembering the upstream homepage & original download location died. The homepage seems to be intermittently alive these days.

We're  using a git tarball because 2.4 was never officially released but er, is the recommended "release" to use. We could check out the commit from the `ffmpeg` website, which would be using the https://git.ffmpeg.org/gitweb/rtmpdump.git/commit/fa8646d commit, but there's no `:tag` for it. Prior discussion in https://github.com/Homebrew/legacy-homebrew/pull/38622.